examples/AvroBlogExamples/Program.cs

```diff
@@ -80,12 +82,14 @@ static void ProduceSpecific(string bootstrapServers, string schemaRegistryUrl)
                             Severity = MessageTypes.LogLevel.Info,
                             Tags = new Dictionary<string, string> { { "location", "CA" } }
                         }
-                    });
+                    },
+                    SerdeType.Regular, SerdeType.Avro);

```

Comment:
the `ProduceAsync` method provided by `Confluent.Kafka.Avro` allows the user to choose whether to use the regular or avro serde for each of key / value.